setup.py

```diff
@@ -24,16 +24,16 @@
 
 setup(
     name="django-google-maps",
-    version='0.10.0',
+    version='0.10.1',
     author="Aaron Madison",
     author_email="aaron.l.madison@gmail.com",
     description="Plugs google maps V3 api into Django admin.",
     long_description=LONG_DESCRIPTION,
     url="https://github.com/madisona/django-google-maps",
     packages=("django_google_maps",),
     include_package_data=True,
-    install_requires=open('requirements/requirements.txt').read().split('\n'),
-    tests_require=open('requirements/test.txt').read().split('\n'),
+    install_requires=open('requirements/requirements.txt').read().split('\n')[:-1],
```

Comment:
I think the better solution is to use `.splitlines()` instead of `split('\n')`. It does not generate any additional record in the list, so slicing is not necessary (also prevents from removing last dependency records if ever `\n` will be not presented at the end).

```
install_requires=open('requirements/requirements.txt').read().splitlines()
```

Reply:
agreed.  @khornberg Thanks for your contribution!  Will you please adjust like @SirZazu suggested?  I've confirmed that it works and is a much cleaner solution.

thanks!

Reply:
Agreed